website/docs/guides/resource-tagging.html.md

```diff
@@ -99,7 +99,7 @@ resource "aws_vpc" "example" {
   }
 
   lifecycle {
-    ignore_tags = [tags["Name"]]
+    ignore_changes = [tags.Name]
```

Comment:
This syntax doesn't seem to be working on terraform 0.12.29. I'm not sure if that's a version problem or if it's not actually supported.